app/models/date_of_birth_change.rb

```diff
@@ -4,7 +4,21 @@ class DateOfBirthChange < ApplicationRecord
 
   delegate :filename, to: :evidence, prefix: true
 
+  enum malware_scan_result: {
+    clean: "clean",
+    error: "error",
+    pending: "pending",
+    suspect: "suspect"
+  }, _prefix: :scan_result
```

Comment:
Nit: There's a neat method you can use when the key and value are the same.

```suggestion
  enum malware_scan_result: %w[clean error pending suspect].index_by(&:itself), _prefix: :scan_result
```